charmcraft/application/main.py

```diff
@@ -40,6 +40,13 @@
     BuildPlannerClass=models.CharmcraftBuildPlanner,
 )
 
+PRIME_BEHAVIOUR_CHANGE_MESSAGE = (
+    "WARNING: the behaviour of the 'prime' keyword has changed in Charmcraft 3. This "
+    "keyword will no longer add files that would otherwise be excluded from the "
+    "charm, instead filtering existing files. Additional files may be added using the "
+    "'dump' plugin. See: https://juju.is/docs/sdk/include-extra-files-in-a-charm"
```

Comment:
```See:```

Since we can't provide link text here, it's better if we provide information about what the user will find/do at this URI (the call to action). So, something like `See [link] for details on how to add files with 'dump'.`, or `... using the 'dump' plugin – see [link] for instructions.` would fit better here.

Reply:
Updated

Reply:
I don't see any new commits.